eng/build.yml

```diff
@@ -168,3 +168,12 @@ phases:
         publishLocation: Container
       continueOnError: true
       condition: always()
+
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/release/7.0.1xx'))) }}:
+  - template: /eng/common/templates/job/onelocbuild.yml
+    parameters:
+      CreatePr: true
+      LclSource: lclFilesfromPackage
+      LclPackageId: 'LCL-JUNO-PROD-DOTNETINSTALLER'
```

Comment:
Is this a project the loc team is aware of❔ We went through https://ceapex.visualstudio.com/CEINTL/_queries/edit/650311/?triage=true to get a new loc package id from the loc team (thanx again @cristianosuzuki77❕).

Reply:
I assume once I have this PR roughly working, I'd have to create an onboarding ticket in the ceapex team project.  @cristianosuzuki77 let me know if I should do that first.